docs/UserGuide.md

```diff
@@ -185,63 +146,6 @@ Deselects the selected event and returns to the global participant list.
 - When no event is selected, `delp 9` deletes the 9th participant completely.
 - `delp 9` after `sel 3` removes the 9th participant from the 3rd event's participant list.
```

Comment:
make evryth revert back to its original formatting

Reply:
😇